healthtools_ke_api/views/sms_handler.py

```diff
@@ -62,6 +84,285 @@ def send_sms(phone_number, msg):
+    # Just incase we found ourselves here with an empty list
+    if len(nurse_list) < 1:
+        return "Could not find a nurse with that name"
+    count = 1
+    msg_items = []
+    for nurse in nurse_list:
+        status = " ".join([str(count) + ".", nurse['name'] +
+                           ",", "VALID TO", nurse['valid_till']])
+        msg_items.append(status)
+        count = count + 1
+    if len(nurse_list) > 1:
+        msg_items.append("Find the full list at http://health.the-star.co.ke")
+
+    return "\n".join(msg_items)
+
+
+def construct_docs_response(docs_list):
+    
+    '''
+        This function builds dynamic message item for a list of doctors supplied by the user
+
+        Args:
+             nurse_list(list): The list of doctors
+        
+        Returns:
+            json. The list of message items for doctors
+    '''
+
+    # Just incase we found ourselves here with an empty list
+    if len(docs_list) < 1:
+        return "Could not find a doctor with that name."
+    count = 1
+    msg_items = []
+
+    for doc in docs_list:
+        # Ignore speciality if not there, dont display none
+        doc = doc['_source']
+        if doc['practice_type'] == "None":
+            status = " ".join([str(count) + ".", "".join(doc['name']), "-",
+                               "".join(doc['reg_no']), "-", "".join(doc['qualifications'])])
+        else:
+            status = " ".join([str(count) + ".", "".join(doc['name']), "-", "".join(doc[
+                                                                                        'reg_no']), "-",
+                               "".join(doc['qualifications']), "".join(doc['practice_type'])])
+        msg_items.append(status)
+        count = count + 1
+    if len(docs_list) > 0:
+        msg_items.append("Find the full list at http://health.the-star.co.ke")
+
+    return "\n".join(msg_items)
+
+
+def clean_query(query):
+    '''
+        This function removes whitespaces, fullstop mark
+
+        Args:
+            query(str): The owrd to clean
+        Returns:
+            str. The lowercase of the cleaned word
+    '''
+
+
+    query = query.lower().strip().replace(".", "")
+    return query
+
+
+def parse_elastic_search_results(response):
+    '''
+        This function builds a list of result set with size as specified in the config variable <SMS_RESULT_COUNT>
+
+        Args:
+            response(json):
+                The search result
+        Returns:
+            The parsed list of response object
+    '''
+
+    result_to_send_count = SMS_RESULT_COUNT
+    data_dict = response.json()
+    fields_dict = (data_dict['hits'])
+    hits = fields_dict['hit']
+    result_list = []
+    search_results_count = len(hits)
+    print "FOUND {} RESULTS".format(search_results_count)
+    for item in hits:
+        result = item
+        if len(result_list) < result_to_send_count:
+            result_list.append(result)
+        else:
+            break
+    return result_list
+
```

Comment:
Why have we added all these functions above?

Reply:
We didn't add the functions.
The functions were added when the `telegram bot` folder was added to the develop branch.

Reply:
@andela-mmakinde If that is correct, then a simple `git pull` should resolve seeing this diff. Otherwise what's happened here is that the functions that live in `build_query.py` have been moved into `sms_handler.py` manually.

Reply:
`git pull` meaning pulling latest changes from `develop` branch.

Reply:
@DavidLemayian We pulled from the develop branch before we pushed and raised the PR.
We pulled a minute ago and our branch is `up to date`.
We share the same shock because we pulled from the develop branch before raising the PR.
We wonder why the changes we pulled from the develop branch did not reflect on the develop branch  in the PR